### DISCUSSION TITLE
Subsequent Devices, Fetch, custom, disabled ? No proper documentation

### DISCUSSION THREAD

**Discussion by slimjet4u** [vrtmrz/obsidian-livesync#434] [Q&A]
Whenever I copy and paste the obsidian settings to another device and enter the password. I am faced with a dialogue asking for 
"Fetch", "Custom", "Disabled"
Sometimes I see "Overwrite" too, this is so confusing. 

Also the plugins from the main device and all struck off and are not enabled in the new device. There is no proper documentation on how to make things work. Please write better structured documentation or process on how to make the customization sync work. And how to reset things if they stop working. 

I found this https://github.com/vrtmrz/obsidian-livesync/blob/main/docs/terms.md documentation.

What to select ? what is it ?

**Comment by MinamiHasaki** [comment]
From your main device, select "Overwrite". From your other devices, select "Fetch".
